### PR TITLE
feat(Sprint Poker): Hitting up and down arrow keys cycles through poker cards

### LINKED ISSUE #6350: Poker: Hitting up down arrow keys should cycle through poker cards
i'm lazy & i don't wanna use my mouse. i wanna just hit up/down arrow keys to select my poker card & then hit the right arrow to go to the next stage

AC
- [ ] when up arrow is tapped, the card to the right is selected
- [ ] if no card is selected, pressing up goes to the left-most card
- [ ] if the right-most card is selected, up goes to the left-most card
- [ ] the opposite happens for pressing the down arrow

Estimate: 3 hours

### PR DESCRIPTION
**Pull request by gitstart-parabol**
# Description

Fixes #6350
I added keyboard event handlers so users can select poker cards using the up and down arrow buttons on the keyboard.

## Demo

https://www.loom.com/share/b7ee91a640534d24935ed4df7660fd64

## Testing scenarios

- [ ] Scrolling right
  - Push the up button
  - If no card was previously selected, this should select the left-most card
  - If the right-most card was previously selected, this should also select the left-most card
  - Otherwise, this should select the next card to the right

- [ ] Scrolling left
  - Push the down button
  - If no card was previously selected, this should select the right-most card
  - If the left-most card was previously selected, this should also select the right-most card
  - Otherwise, this should select the next card to the left

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/components/PokerCardDeck.tsx [hunk 1]
```diff
@@ -185,6 +191,35 @@ const PokerCardDeck = (props: Props) => {
...
+        // Select the right-most card if no card is selected or if the left-most card was previously selected
+        vote((cards[cards.length - 1] as Card).label)
+      } else {
+        // Otherwise select the card to the left
+        vote((cards[selectedIdx - 1] as Card).label)
+      }
+    }
+  }
+
+  window.onkeydown = onKeyDown
```
Comment (mattkrick) [inline packages/client/components/PokerCardDeck.tsx:221]: -1 we'll want to clean up the function on unmount. Consider using https://github.com/ParabolInc/parabol/blob/f3361ab138d76ecbd2fa233e518790226414709e/packages/client/hooks/useHotkey.ts#L7

**Review by mattkrick — CHANGES_REQUESTED** [review]

File: packages/client/components/PokerCardDeck.tsx (hunk 1 above)
Comment (gitstart-parabol) [reply packages/client/components/PokerCardDeck.tsx]: Done @mattkrick

**Comment by gitstart-parabol** [comment]
@JimmyLv @mattkrick this PR is ready for review

**Review by JimmyLv — APPROVED** [review]

**Comment by gitstart-instrumental** [comment]
@mattkrick this PR is awaiting your review

**Comment by gitstart-parabol** [comment]
@mattkrick this PR is awaiting your review

**Review by Dschoordsch — CHANGES_REQUESTED** [review]
Looks ok and works. Only too many comments which make the code hard to read.

**Comment by gitstart-parabol** [comment]
> Looks ok and works. Only too many comments which make the code hard to read.

@Dschoordsch thanks for the feedback. The comments have been removed, please check again

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**